core/src/test/scala/neotypes/CypherQueryInterpolationSpec.scala

```diff
@@ -141,10 +141,35 @@ final class CypherQueryInterpolationSpec extends AnyFlatSpec {
 
     assert(query.query == expected)
   }
+
+  it should "interpolation with case classes and a relationship" in {
+    val user = User("Joan", 20)
+    val cat = Cat("Waffles", 3)
+    val relationship = HasCatRelationship(2)
+
+    val query = c"CREATE (u: User { $user }) -[r:HAS_CAT { $relationship }]->(c:Cat { $cat }) RETURN r"
```

Comment:
:+1:

Now the only test that would be missing would be something like: `c"CREATE (u: User { $user, extraProperty: $foo }"`

Reply:
done